pom.xml

```diff
@@ -443,6 +458,17 @@
     </pluginManagement>
     <plugins>
       <plugin>
+        <artifactId>maven-jar-plugin</artifactId>
+        <executions>
+          <execution>
+            <goals>
+              <goal>jar</goal>
+              <goal>test-jar</goal>
```

Comment:
these need to be in separate executions if you want to have a separate skip flag